docker/common.ps1

```diff
@@ -35,15 +35,19 @@ function GetUbuntuImageTags($repo, $version, $arch) {
     switch ($arch) {
         "x64" { 
             return @(
-                "$($repo):$($version)-ubuntu.20.04-x64",
-                "$($repo):5.2-ubuntu-latest"
+                "$($repo):latest",
+                "$($repo):latest-lts",
+                "$($repo):ubuntu-latest",
```

Comment:
should we consider `ubuntu-latest-lts` ?

Reply:
How about `ubuntu-lts`?